App.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8"/>
```

Comment:
Why is this project running on .NETFramework?

Reply:
There are issues with the environment, but I updated for the next projects.